src/test/kotlin/io/bazel/kotlin/builder/tasks/BUILD.bazel

```diff
@@ -19,6 +19,15 @@ java_library(
     name = "JdepsParserTestFixtures",
     testonly = True,
     srcs = glob(["testFixtures/*.java"]),
+    deps = [
+        ":JdepsParserTestFixtures2",
+    ],
+)
+
+java_library(
+    name = "JdepsParserTestFixtures2",
+    testonly = True,
+    srcs = glob(["testFixtures2/*.java"]),
```

Comment:
Might look less copy-paste grouping these different test figures inside directories within the `testFixtures` directory instead of just duplicating and adding a numerical index. Ex: `testFixtures/a-sources/` `testFixtures/b-sources/`